src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const transformState = [state];
```

Comment:
in this way it would be better, you will change the clone and push it to state in the end of switch
```suggestion
  const state = []
  const clone= {...state};
```

Reply:
why redefine the state?

Reply:
This is to make a copy of the state, because working with the parameter directly is not a good idea

Reply:
if i make these changes i get an error : 10:9  error  Parsing error: Identifier 'state' has already been declared